proto/types/http.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+package sdk.types;
+
+option go_package = "github.com/pokt-network/shannon-sdk/types";
+
+// POKTHTTPRequest represents an http.Request to be serialized and sent to a RelayMiner
+// by an Application/Gateway client within a RelayRequest payload.
+message POKTHTTPRequest {
+  string method = 1;
```

Comment:
#PUC Is this a REST method?

Reply:
If it is a REST call, this would be the method (GET, POST, ...). If it's a JSONRPC call it would be just as the original request (should be `POST`)